helm/values-prod.yaml

```diff
@@ -61,13 +61,48 @@ microservice-chart:
           type: Utilization # Allowed types are 'Utilization' or 'AverageValue'
           value: "75"
   envConfig:
-    WEBSITE_SITE_NAME: "pagopa<project-name>" # required to show cloud role name in application insights # TODO
+    WEBSITE_SITE_NAME: "pagopareceiptpdfnotifier" # required to show cloud role name in application insights
     FUNCTIONS_WORKER_RUNTIME: "java"
+    NOTIFIER_QUEUE_TOPIC: "pagopa-d-weu-receipts-queue-receipt-io-notifier-error"
+    COSMOS_RECEIPT_SERVICE_ENDPOINT: "https://pagopa-d-weu-receipts-ds-cosmos-account.documents.azure.com:443/"
```

Comment:
attention to use DEV var secret into PROD env

Reply:
Changed to correct values